### PR TITLE
Adding Portuguese (pt-BR) translation

### LINKED ISSUE #298: Portuguese translation

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #298 

## Description
This PR is replacing the [original PR](https://github.com/telosnetwork/teloscan/pull/305) created by [7glaucio](https://github.com/7glaucio) which needed to be updated because of the last structural modifications. It is based on the last version of the English file (in file `src/i18n/en-us/index.js`.)

### CODE REVIEW

**Comment by Viterbo** [comment]
This PR is blocked too until [PR292](https://github.com/telosnetwork/teloscan/pull/292) is actually merged.

**Review by donnyquixotic — APPROVED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Review by donnyquixotic — APPROVED** [review]

File: src/i18n/pt-br/index.js [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+            too_large: 'O valor máximo para int{ size } é { max }',
+            too_small: 'O valor mínimo para int{ size } é { max }',
+            too_large_pow2: 'O valor máximo para uint{ size } é 2^{ size } - 1',
+            too_small_pow2: 'O valor mínimo para int{ size } é -(2^{ size }) + 1',
+            too_large_unsigint: 'O valor máximo para uint{ size } é 2^{ size } - 1',
+            no_negative_unsigint: 'O valor para uint{ size } não deve ser negativo',
+            invalid_signed_integer: 'Inteiro assinado inválido',
+            invalid_unsigint: 'A entrada deve ser um inteiro não assinado válido',
+            str_input_placeholder: '["algum valor", ... , "valor final"]',
+            str_input_hint: 'Aspas duplas em strings (sequência de textos) devem ser evitadas(\\")',
```
Comment (karynemayer) [inline src/i18n/pt-br/index.js:279]:
```suggestion
            str_input_hint: 'Aspas duplas em strings (sequência de caracteres) devem ser evitadas(\\")',
```

File: src/i18n/pt-br/index.js [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+            address_placeholder: 'Endereço que começa com 0x',
+            address_label: '{ label } (endereço)',
+            boolean_array_label: '{ label } (bool[{ size }])',
+        },
+        health: {
+            status: 'Estado',
+            checked_at: 'Verificado em',
+            task: 'Tarefa',
+            message: 'Mensagem',
+            block_height: 'Altura do bloqueio',
```
Comment (karynemayer) [inline src/i18n/pt-br/index.js:289]:
```suggestion
            block_height: 'Altura do bloco',
```

File: src/i18n/pt-br/index.js [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+            health_status: 'Estado de saúde',
+            advanced: 'Avançado',
+            light_mode: 'Modo claro',
+            dark_mode: 'Modo escuro',
+            address_not_found: 'A busca pelo endereço EVM vinculado à conta nativa { account } falhou.',
+            search_failed: 'A busca falhou, insira um termo de pesquisa válido.',
+            goto_address_details: 'Ir para detalhes do endereço',
+            copy_address: 'Copiar endereço',
+            address_copied: 'Endereço copiado para a área de transferência',
+            search_placeholder: 'Endereço, Tx, Bloqueio',
```
Comment (karynemayer) [inline src/i18n/pt-br/index.js:340]:
```suggestion
            search_placeholder: 'Endereço, Tx, Bloco',
```

**Review by karynemayer — CHANGES_REQUESTED** [review]
I left a few suggestions on wording that I thought it would make sense.

File: src/i18n/pt-br/index.js [hunk 4]
```diff
@@ -0,0 +1,326 @@
+export default {
+    pages: {
+        staking: {
+            note_unstaking_period: 'Lembre-se que há um período de retirada de {period}',
```
[the remark this answers is not in the document]
Comment (Viterbo) [reply src/i18n/pt-br/index.js:4]: The period value comes from a call to this function which already manages translation:

https://github.com/telosnetwork/teloscan/blob/dev/src/pages/staking/staking-utils.js#L10-L32
```typescript
export function formatUnstakePeriod(seconds, $t) {
    if (seconds === null)
        return '--';

    let quantity;
    let unit;

    if (seconds < HOUR_SECONDS) {
        quantity = seconds / 60;
        unit = $t('pages.staking.minutes');
    } else if (seconds < DAY_SECONDS) {
        quantity = seconds / HOUR_SECONDS;
        unit = $t('pages.staking.hours');
    } else {
        quantity = seconds / DAY_SECONDS;
        unit = $t('pages.staking.days');
    }

    if (!Number.isInteger(quantity))
        quantity = quantity.toFixed(1);

    return `${quantity} ${unit}`;
}
```

**Review by donnyquixotic — APPROVED** [review]

**Review by donnyquixotic — DISMISSED** [review]

File: src/i18n/pt-br/index.js [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+    components: {
+        verify_prompt: 'Este contrato ainda não foi verificado. Você gostaria de carregar o(s) contrato(s) e os metadados para verificar o código-fonte agora?',
+        verify_contract: 'Verificar contrato',
+        search_evm_address_failed: 'A busca pelo endereço EVM vinculado à conta nativa { accountName } falhou. Você pode criar uma em wallet.telos.net',
+        unknown_web3_login_type: 'Tipo de login web3 desconhecido: { provider }',
+        connect_wallet: 'Conectar carteira',
+        view_address: 'Ver endereço',
+        disconnect: 'Desconectar',
+        disable_wallet_extensions: 'Desative as carteiras de extensão ou defina a Brave Wallet como padrão nas configurações da carteira do navegador para usar a Brave Wallet',
+        enable_wallet_extensions: 'Ative a extensão MetaMask e defina o padrão como `preferir extensões` nas configurações da carteira do navegador para usar a carteira MetaMask',
```
Comment (Viterbo) [inline src/i18n/pt-br/index.js:195]: @karynemayer, these two lines were recently added in the [Brave native wallet PR](https://github.com/telosnetwork/teloscan/pull/314). 

Can you check them and suggest corrections if needed?

**Review by karynemayer — APPROVED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**